R/tm_g_distribution.R

```diff
@@ -172,6 +170,9 @@ tm_g_distribution <- function(label = "Distribution Module",
   )
 }
 
+# UI function for the distribution module.
+#' @noRd
+#' @keywords internal
```

Comment:
Let's resolve the decision on how we should be using the `@noRd` and `@keywords internal` tags.
The previous 3 PRs that I reviewed did not add any tags.

Reply:
yes, this should be removed